include/roboteam_ai/control/controllers/BaseController.h

```diff
@@ -0,0 +1,25 @@
+//
+// Created by ratoone on 25-09-19.
+//
+
+#ifndef RTT_BASECONTROLLER_H
+#define RTT_BASECONTROLLER_H
+
+template <typename Input, typename Output>
+/**
+ * The base abstract class for the controllers. Any control method should inherit from this
+ * @tparam Input
+ * @tparam Output
```

Comment:
The @param tag should not be used here, because the class does not use parameters only the function uses parameter.